app/controllers/application_controller.rb

```diff
@@ -1,8 +1,21 @@
 class ApplicationController < ActionController::Base
+  include Response
+  include ExceptionHandler
```

Comment:
The following error is preventing the project to run. Please clone your project go through the process tp run the project and make sure there is no running error.
![exception](https://user-images.githubusercontent.com/35504149/146959258-89346879-38c0-4a1d-85c1-a2e0c9b1b640.png)